warehouse/query-core/src/test/java/datawave/query/PushdownQueryTest.java

```diff
@@ -138,6 +138,21 @@ public void testDelayedFilterIncludeRegex() throws Exception {
         }
     }
     
+    @Test
+    public void testDelayedFilterIncludeRegexExpansion() throws Exception {
+        log.info("------  testErrorFilterIncludeRegexExpansion  ------");
```

Comment:
wrong test name in log message?

Reply:
This is a silly reason, but the ticket explicitly says to "Duplicate testErrorFilterIncludeRegex but do not disable expansion." in PushdownQueryTest, which only matches the info statement and not the actual method, so I tried to make it clear how the two were related by maintaining the same level of wrongness between method name and info statement.  I'll fix them both.